### PR TITLE
Fix path separators in .kps files

### PR DESCRIPTION
**Pull request by darcywong00**
Fixes #3 so lexical models can be built on other platforms

I'm not updating model versions since the compiled artifacts should be the same (files are in a flat structure inside the .kmp)

### CODE REVIEW

File: release/example/crk.wordlist_wahkohtowin/source/example.crk.wordlist_wahkohtowin.model.kps [hunk 1]
```diff
@@ -15,7 +15,7 @@
   </Info>
   <Files>
     <File>
-      <Name>..\build\example.crk.wordlist_wahkohtowin.model.js</Name>
```
Comment (mcdurdin) [inline release/example/crk.wordlist_wahkohtowin/source/example.crk.wordlist_wahkohtowin.model.kps]: According to the .kps spec, these should be backslashes. This is not ideal, but it is what it is. The compiler needs to transform them instead.

Comment (darcywong00) [reply release/example/crk.wordlist_wahkohtowin/source/example.crk.wordlist_wahkohtowin.model.kps]: alright, I added a regex for the slashes

**Comment by eddieantonio** [comment]
@darcywong00 Does this fix #23, not #3?

**Comment by darcywong00** [comment]
My bad. I've fixed the associated issue number.

**Pull request merged**